dlc/src/main/scala/org/bitcoins/dlc/BinaryOutcomeDLCWithSelf.scala

```diff
@@ -183,21 +184,48 @@ case class BinaryOutcomeDLCWithSelf(
   }
 
   def createFundingTransaction: Future[Transaction] = {
-    val output: TransactionOutput =
-      TransactionOutput(totalInput, P2WSHWitnessSPKV0(fundingSPK))
-    val localChange =
-      TransactionOutput(Satoshis(localFunding) - localInput, localChangeSPK)
-    val remoteChange =
-      TransactionOutput(Satoshis(remoteFunding) - remoteInput, remoteChangeSPK)
-
-    val outputs: Vector[TransactionOutput] =
-      Vector(output, localChange, remoteChange)
-    val txBuilderF: Future[BitcoinTxBuilder] =
-      BitcoinTxBuilder(outputs, fundingUtxos, feeRate, emptyChangeSPK, network)
-
-    txBuilderF.flatMap { txBuilder =>
-      subtractFeeFromOutputsAndSign(txBuilder,
-                                    Vector(localChangeSPK, remoteChangeSPK))
+
+    val emptyOutPoint =
+      TransactionOutPoint(DoubleSha256DigestBE.empty, UInt32.zero)
+    val dummySpendingInfo = P2WSHV0SpendingInfo(
+      outPoint = emptyOutPoint,
+      amount = totalInput * 2,
+      scriptPubKey = P2WSHWitnessSPKV0(fundingSPK),
+      signers = Vector(fundingLocalPrivKey, fundingRemotePrivKey),
+      hashType = HashType.sigHashAll,
+      scriptWitness = P2WSHWitnessV0(fundingSPK),
+      conditionalPath = ConditionalPath.NoConditionsLeft
+    )
+    val cetWinLocalF = createCETWinLocal(dummySpendingInfo)
```

Comment:
There is symmetry between `cetWinLocal` and `cetLoseRemote` right? I guess to me it seems that this scheme is not going to work when parties are trying to "come to consensus" on what fees are. There are 2 different outcomes in the binary DLC case, is there a reason you picked `CETWinLocal` as the one to estimate fees on and not `CETLoseLocal`? Comments are helpeful here.

Reply:
In my testing, all of the CETs had the same fees since they are all the same size (since they are all the same contract). Refund is a couple sats fewer as it is shorter. I'll add a comment explaining this though. (also keep in mind this is a fee estimation, we're almost never gonna get it right on the dot so if there's like a couple bytes difference this should be fine)

Reply:
Also to your point about fee consensus, the initiator will specify fees over messages in the future. Or something of this nature.